garrysmod/gamemodes/base/entities/entities/base_anim.lua

```diff
@@ -35,9 +35,11 @@ end
 
 if ( CLIENT ) then
 
+	local Entity = FindMetaTable( "Entity" )
+
 	function ENT:Draw( flags )
 
-		self:DrawModel( flags )
+		Entity.DrawModel( self, flags )

```

Comment:
Isn't it easier to do `ENT.Draw = Entity.DrawModel`?

Reply:
Yeah, you're right. Thanks.